File: resources/c-sharp/multiple-return-statements.cs
public class MultipleReturnStatements {

    public int function_with_more_than_one_return(int x) {
        if (x) {
            return 2;
        }
        return 1;
    }

    private int function_with_only_one_return() {
        return 3;
    }

}

File: resources/php/multiple-return-statements.php
<?php

/**
 * Complexity test resource file
 */

namespace Declarations;

class MultipleReturnStatements {

    public function function_with_more_than_one_return($x, $y): int {
        if ($x) {
            return 2;
        }
        return 1;
    }

    private function function_with_only_one_return(): int {
        return 2;
    }
}